### PR TITLE
network: mention feature availability in the help

### PR DESCRIPTION
**Pull request by thc202**
Add notes in the help indicating in which versions are the features
available.
Restructure the changelog for clarity.

### CODE REVIEW

File: addOns/network/CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
-- Server certificates management on newer ZAP versions.
-- Handle command line arguments `-certload`, `-certpubdump`, and `-certfulldump` on newer ZAP versions.
-- Options panel to manage the root CA certificate and issued certificates on newer ZAP versions.
-- API endpoints to generate, import, obtain, and to configure the validity of the root CA
-certificate and issued certificates (Issue 4673).
+- API endpoints to generate, import (Issue 2280), and obtain the root CA certificate.
```
Comment (ricekot) [inline addOns/network/CHANGELOG.md:9]: Can link to Issue 2280 `https://github.com/zaproxy/zaproxy/issues/2280`.

File: addOns/network/CHANGELOG.md [hunk 2]
```diff
@@ -6,8 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- On weekly and versions after 2.11:
```
Comment (ricekot) [inline addOns/network/CHANGELOG.md:10]:
```suggestion
- On weekly releases and versions after 2.11:
```
Optionally - can be more specific, e.g.: `On weekly releases (after Dec 3, 2021) and versions after 2.11:`

File: addOns/network/src/main/javahelp/help/contents/options/servercertificates.html [hunk 3]
```diff
@@ -6,6 +6,8 @@
 </HEAD>
 <BODY>
 	<H1>Server Certificates</H1>
+	<strong>Note:</strong> This feature is only available in weekly releases and versions after 2.11.
+	<p>
```
Comment (ricekot) [inline addOns/network/src/main/javahelp/help/contents/options/servercertificates.html:10]: This `<p>` tag is not closed (same for other additions).

Comment (thc202) [reply addOns/network/src/main/javahelp/help/contents/options/servercertificates.html:10]: Does not need to, can replace with a `br`.

**Comment by thc202** [comment]
Changes done.

**Review by ricekot — APPROVED** [review]
Thank you!

**Review by kingthorin — COMMENTED** [review]
Thanks

**Review by kingthorin — APPROVED** [review]

**Pull request merged**